packages/malloy/src/dialect/functions/length.ts

```diff
@@ -41,6 +41,7 @@ export function fnLength(): DialectFunctionOverloadDef[] {
   ];
 }
 
+// TODO: add support for byte length in postgres, duckdb
```

Comment:
There should be support for these today, defined in `/src/dialect/postgres/functions/byte_length.ts` and `/src/dialect/duckdb/functions/byte_length.ts`.